common/components/data/_data.scss

```diff
@@ -8,22 +8,39 @@
   @include govuk-font($size: 36, $weight: bold);
   display: block;
   margin: 0;
+
+  .govuk-tag {
+    position: relative;
+    top: -6px;
+  }
 }
 
 .app-data--s {
   .app-data__value {
     @include govuk-font($size: 24, $weight: bold);
+
+    .govuk-tag {
+      top: -2px;
+    }
   }
 }
 
 .app-data--l {
   .app-data__value {
     @include govuk-font($size: 48, $weight: bold);
+
+    .govuk-tag {
+      top: -10px;
+    }
   }
 }
 
 .app-data--xl {
   .app-data__value {
     @include govuk-font($size: 80, $weight: bold);
+
+    .govuk-tag {
+      top: -22px;
```

Comment:
I assume there isn't a nice govuk variable we could be using for this?

Reply:
I don't _think_ so. I added these to correct the position as the label is always a fixed font size, so if they're ever used with other data sizes (we don't currently have a use case ourselves) the position of the label won't look too out of place.